src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/documents/DocumentApiClientTest.kt

```diff
@@ -49,34 +49,47 @@ class DocumentApiClientTest {
 
   @Test
   fun `Returns the document response when successfully uploading a document to the remote service`() {
-    stubUploadDocumentEndpoint(status = 201, responseBody = happyResponse)
+    givenDocumentApiRespondsWith(status = 201, responseBody = happyResponse)
 
-    val result = documentApiClient.uploadDocument(
-      file = file.contentAsByteArray,
-      documentUuid = UUID.fromString("e2487a03-7cf9-4a9c-85e4-1d51efd7b3f1"),
-      documentType = DocumentType.EXCLUSION_ZONE_MAP,
-      metadata = mapOf("aKey" to "1", "bKey" to "2", "cKey" to "3"),
-    )
+    val result = uploadDocument()
 
-    assertEquals(result?.documentUuid, UUID.fromString("e2487a03-7cf9-4a9c-85e4-1d51efd7b3f1"))
-    assertEquals(result?.documentType, DocumentType.EXCLUSION_ZONE_MAP)
+    assertEquals(result.documentUuid, UUID.fromString("e2487a03-7cf9-4a9c-85e4-1d51efd7b3f1"))
```

Comment:
we tend to use assertJ assertions rather than the native junit.. leads to slightly nicer error messages and it's easier to just statically import a single thing which is then contextual based on the passed in type. 
`assertThat(1).isGreaterThan(2)` as it's number
`assertThat("bob").contain("ob")` as it's a string.